components/TARGET_PSA/services/inc/mbed_spm_partitions.h

```diff
@@ -0,0 +1,307 @@
+/* Copyright (c) 2017-2019 ARM Limited
```

Comment:
Is this useful to TF-M builds? Should we place it under `components/TARGET_PSA/TARGET_MBED_SPM/inc/`?

Reply:
Looks Mbed services use this, even when TF-M SPM is used. OK